proxies/live/apiproxy/targets/apim.xml

```diff
@@ -44,18 +44,6 @@
           <Condition>(access-mode Equals "user-restricted")</Condition>
         </Step>
         <!-- Get ODS code from UserInfo -->
-        <Step>
-          <Name>JavaScript.GetUserDetailsFromCache</Name>
-          <Condition>(access-mode Equals "user-restricted")</Condition>
-        </Step>
-        <Step>
-          <Name>AssignMessage.AddUserInfoHeaders</Name>
```

Comment:
should we look in the eps coordinator to find where these headers were consumed? probs needs deleting around there somewhere too